ui/intermittent-failures/GraphsContainer.jsx

```diff
@@ -97,20 +97,18 @@ GraphsContainer.propTypes = {
     }),
   ),
   graphTwoData: PropTypes.arrayOf(
-    PropTypes.arrayOf(
-      PropTypes.shape({
-        date: PropTypes.shape({ Date: PropTypes.string }),
-        value: PropTypes.number,
-      }),
-    ),
+    PropTypes.shape({
+      date: PropTypes.shape({ Date: PropTypes.string }),
+      value: PropTypes.number,
+    }),
     PropTypes.arrayOf(
       PropTypes.shape({
         date: PropTypes.shape({ Date: PropTypes.string }),
         value: PropTypes.number,
       }),
     ),
   ),
-  children: PropTypes.object.isRequired,
+  children: PropTypes.element.isRequired,
```

Comment:
hrm, I think this should be the same as `datePicker`, which is passed into `GraphsContainer` as the child prop (in the `Layout` component):
`  children: PropTypes.oneOfType([PropTypes.shape({}), PropTypes.bool])` and not required.

Reply:
Okay, I will confirm this too. Thanks

Reply:
It seems to be a `DateRangePicker` component. That's why I used the `PropTypes.element` prop type.

![image](https://user-images.githubusercontent.com/36057474/79806228-d52f4c80-835f-11ea-90fa-070162c0831f.png)

Should I use `PropTypes.element` with a default prop of `children: null,` instead or  If I use `children: PropTypes.oneOfType([PropTypes.shape({}), PropTypes.bool])` can I use `children: null,` or `children: false,` as default prop, since default prop is required in this case?

Reply:
@sarah-clements I left some comments

Reply:
Either of those will work, I think. Whatever you choose though, let's keep it consistent with how the datePicker prop is being used in the GraphsContainer file.

It seems like the purpose of `children: PropTypes.element` is to make it clear that only one child prop is allowed. This was not necessarily the intention but it's probably ok. Using `PropTypes.shape({})` is also ok, since any instances of class components or functional components are ultimately objects.